gitpod-io/gitpod
# disable VS Code Desktop for stable

#### What it does

VS Code Desktop is not ready yet to be available for everyone. This PR enables it only on latest.

Changes in Gitpod Code: https://github.com/gitpod-io/vscode/commit/330c9b35c2f3d95af9192bc1ce23eccc5fcbe808

#### How to test

- Test stable: https://ak-no-code-desktop-in-stable.staging.gitpod-dev.com/#github.com/akosyakov/parcel-demo
  - in About dialog it should be 1.59.0, but in File menu there should NOT be open in VS Code
- Switch to latest in settings: https://ak-no-code-desktop-in-stable.staging.gitpod-dev.com/preferences
- Test latest: https://ak-no-code-desktop-in-stable.staging.gitpod-dev.com/#github.com/akosyakov/parcel-demo
  - in About dialog it should be 1.59.0, but in File menu there should be open in VS Code
